src/Index.php

```diff
@@ -36,7 +40,7 @@ public function index() {
         
         $content .= '</ol>';
         
-        require 'layout.phtml';
+        require 'templates/layout.phtml';
```

Comment:
I typically wouldn't put my template directory inside my src/ directory; I'd leave that for namespaced classes only. Up to you though.